.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -10,28 +10,26 @@ body:
         You'll need to demonstrate widespread support for your idea among the community.
 
         Major feature proposals should generally be discussed on [Discourse](https://discuss.python.org/c/ideas/6) before opening a GitHub issue. Wait until it's clear that most people support your idea before filling in this form.
-  - type: checkboxes
+  - type: dropdown
     attributes:
-      label: Has this already been discussed elsewhere?        
+      label: Has this already been discussed elsewhere?
       options:
-        - label: I have already discussed this feature proposal on Discourse
-        - label: This is a minor feature, which does not need previous discussion elsewhere
+        - No response given
```

Comment:
The schema for [dropdown](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-for-dropdown) says ``None`` can't be given if default is set, so I wonder if it would enforce choosing one of the two options? Untested:

```suggestion
        - None
```

Reply:
Let's try it out!

Reply:
![image](https://github.com/python/cpython/assets/9087854/d4c1f92a-8861-47ac-906c-087b6bd6b054)

Oh dear...

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#bodyi-options-must-not-include-the-reserved-word-none

Reply:
That's a "no" from GitHub :(

> There is a problem with this template
> 
> body[1]: options must not include the reserved word, 'None'.

![image](https://github.com/python/cpython/assets/66076021/78150a09-3af9-4516-9218-6a4304add554)

Reply:
Yeah, it's somewhat frustrating -- nothing quite fits what we really want here